cristin-import/src/main/java/no/unit/nva/cristin/mapper/CristinBookOrReportMetadata.java

```diff
@@ -20,18 +20,18 @@
 )
 @AllArgsConstructor(access = AccessLevel.PACKAGE)
 @JsonIgnoreProperties({"hefte", "utgave", "utgave_fra", "doi", "status_revidert", "status_elektronisk_publisert",
-        "status_utgitt_av_forlag", "stedangivelse_utgiver", "landkode_utgiver", "institusjonsnr_utgiver",
-        "avdnr_utgiver", "undavdnr_utgiver", "gruppenr_utgiver", "tidsskriftnr_serie", "volum_serie",
-        "sprakkode_oversatt_fra", "sprakkode_oversatt_til", "originalforfatter", "originaltittel",
-        "forlag", "tidsskrift_serie", "arkivpost"})
-public class CristinBookReport {
+    "status_utgitt_av_forlag", "stedangivelse_utgiver", "landkode_utgiver", "institusjonsnr_utgiver",
+    "avdnr_utgiver", "undavdnr_utgiver", "gruppenr_utgiver", "tidsskriftnr_serie", "volum_serie",
+    "sprakkode_oversatt_fra", "sprakkode_oversatt_til", "originalforfatter", "originaltittel",
+    "forlag", "tidsskrift_serie", "arkivpost"})
+public class CristinBookOrReportMetadata {
```

Comment:
This class represents objects that appear under the name "type_bok_rapport" and contains information that is found in books and reports. Renamed for better understanding